src/applications/pensions/config/form.js

```diff
@@ -1073,21 +1082,26 @@ const formConfig = {
                   'ui:title': 'Does your child live with you?',
                   'ui:widget': 'yesNo',
                 },
-                childAddress: merge(
-                  {},
-                  address.uiSchema(
-                    'Address',
-                    false,
-                    (form, index) =>
-                      !get(['dependents', index, 'childInHousehold'], form),
-                  ),
-                  {
-                    'ui:options': {
-                      expandUnder: 'childInHousehold',
-                      expandUnderCondition: false,
+                childAddress: {
+                  ...addressUI({
```

Comment:
can just do `childAddress: addressUI({...` here, since it returns a new object

Reply:
the return value of the `addressUI` call needs to be spread into a new object with the `'ui:options'` key.